packages/js/vite.config.js

```diff
@@ -2,19 +2,31 @@ import { resolve } from "path";
 import { defineConfig } from "vite";
 import dts from "vite-plugin-dts";
 
-export default defineConfig({
-  build: {
-    emptyOutDir: false, // keep the dist folder to avoid errors with pnpm go when folder is empty during build
-    minify: "terser",
-    sourcemap: true,
-    lib: {
-      // Could also be a dictionary or array of multiple entry points
-      entry: resolve(__dirname, "src/index.ts"),
-      name: "formbricks",
-      formats: ["cjs", "es", "umd", "iife"],
-      // the proper extensions will be added
-      fileName: "index",
+const config = ({ mode }) => {
+  const isDevelopment = mode === "dev";
+  const formbricksSurveysScriptSrc = isDevelopment
+    ? "http://localhost:3003/index.umd.js"
+    : "https://unpkg.com/@formbricks/surveys@^1.4.0/dist/question-date.umd.js";
```

Comment:
@pandeymangg I guess this should also be `index.umd.js`, right? (instead of `question-date.umd.js`)
Can we also pull the current version from the package json and this way create an URL like:
```
https://unpkg.com/@formbricks/surveys@^1.4/dist/index.umd.js
```
with `1.4` being the first part of the version number (without specifying the path release). Alternatively we can also pull the exact version from the `package.json` in `../surveys/package.json`?